scripts/usb_scan.py

```diff
@@ -374,10 +375,15 @@ def check_hex_length(self, name, value):
         return False
 
     @staticmethod
-    def log_parse_error(pos, end, target, line):
-        log.error(
-            "Malformed policy rule, unable to parse '{}'. Ignored line: {}"
-            .format(target[pos:end], line))
+    def log_exit(m):
+        log.error(m)
```

Comment:
In the case of an error, is the error logged twice? Here, and when the exception is handled? Is using `sys.exit` with a message rather than an exit code idiomatic (I'm not a Python programmer).

Reply:
Comment message updated.

Reply:
And yes, 
the "log.error(m" will put log to /var/log/user.log
the sys.exit(m) will put log to stderr, so that xapi can get it.
Is it OK what we just call exit(1)?

Reply:
I think it's better to keep it as is through the log will print twice. The first log is printed by `log.error` and output to `user.log`, and the second log is printed by `xapi` when catching `SystemExit exception` from the script, and the log will output to `xensource.log`. I think it will make it easier for us to track the log between `user.log` and `xensource.log` when debugging.

Reply:
I agree with @krizex in this case. I'd leave it as is but I would probably move the function to the main scope, next to the log list. 

@lindig yes, using `sys.exit("message...")` is the idiomatic way to print the message on stderr and exit with 1.

Reply:
Move the function to the top level will be good, so we can call it everywhere.